index.html

```diff
@@ -281,6 +281,11 @@ <h4>Game streaming</h4>
              latency for audio, video and data under varying network conditions. This may
              include tweaking of transport parameters for both media and data.</td>
         </tr>
+	<tr>
+           <td>N36</td>
+           <td>An application that is only receiving but not sending media can operate
+	       without prompts for camera and microphone permissions.</td>
```

Comment:
Agreed. If you don't want camera access you shouldn't have to ask for it as a workaround to APIs lacking in other areas.